docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md

```diff
@@ -177,7 +177,29 @@ vbc Example1.vb
 ```  
 Bon jour!  
 ```  
+
+### .NET Core resource fallback process
+ The .NET Core resource fallback process involves the following steps:  
   
+1.  The runtime checks the directory of the currently executing assembly for a directory that matches the requested culture. If it finds the directory, it searches that directory for a valid satellite assembly for the requested culture. The runtime then searches the satellite assembly for the requested resource. If it finds the resource in the assembly, it uses it. If it doesn't find the resource, it continues the search.  
+  
+2.  The runtime raises the <xref:System.Runtime.Loader.AssemblyLoadContext.Resolving?displayProperty=nameWithType> event to indicate that it is unable to find the satellite assembly. If you choose to handle the event, your event handler can return a reference to the satellite assembly whose resources will be used for the lookup. Otherwise, the event handler returns `null` and the search continues.  
```

Comment:
I haven't checked the code, but I'm pretty sure the runtime issues a normal "bind-by-name" - basically `Assembly.Load` equivalent. Which would trigger entire series of things (`AssemblyLoadContext.Load`, then fallback to default, then the default's and custom ALC's `Resolving` event and finally the `AppDomain` resolve event).
This also includes the existing search in the resource probing paths as specified by the host (based on `.deps.json`).

Reply:
Most of what you say is true.  Based on one of our earlier discussions, I had try to avoid unnecessary complexity here.  I over simplified.  Partially because I was focused on pruning the .NET Framework algorithm.

The code is calling into AssemblyNative::Load().  The arguments for satellite load are different than a normal assembly load so the same full normal algorithm is not called.  I will step through the native code to check some of your assertions to make sure this description is as accurate and simple to understand as possible.